system/I18n/Time.php

```diff
@@ -479,21 +479,7 @@ public function getQuarter(): string
      */
     public function getDst(): bool
     {
-        // grab the transactions that would affect today
-        $start       = strtotime('-1 year', $this->getTimestamp());
-        $end         = strtotime('+2 year', $start);
-        $transitions = $this->timezone->getTransitions($start, $end);
-
-        $daylightSaving = false;
-
-        foreach ($transitions as $transition) {
-            if ($transition['time'] > $this->format('U')) {
-                $daylightSaving = (bool) ($transition['isdst'] ?? $daylightSaving);
-                break;
-            }
-        }
-
-        return $daylightSaving;
+        return (bool) $this->format('I');
```

Comment:
Very nice with the `I` flag. But I would like this to compare against it return values for the benefit of myself and others not aware of the flag.
```suggestion
        return $this->format('I') === '1'; // 1 if Daylight Saving Time, 0 otherwise.
```